setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
NetBSD env is located at /usr/bin/env rather than /usr/pkg/bin/env

Reply:
You will find the same misstake a couple of times more. I was unsure about that path, but couldn't check as I was on Void while pushing.

Aside from env, was the rest ok?